### PR TITLE
Tweak query syntax again for populating value on GH

### PR DESCRIPTION
**Pull request by JMHReif**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -35,8 +35,15 @@ image::{model}[]
...
 
+:param-name: location
+:param-value: Iceland, Rekjavik
+
 .Example Query:
-[source,cypher,role=query-example,param-name=location,param-value="Iceland, Rekjavik",result-column=ip,expected-result=10.0.0.254]
+[source,cypher,role=query-example-params]
+----
+:param {param-name} => "{param-value}"
+----
```
Comment (fbiville) [inline README.adoc:45]: you might need a newline in there

File: README.adoc [hunk 2]
```diff
@@ -35,8 +35,15 @@ image::{model}[]
 .Example
 image::{example}[width=600]
 
+:param-name: location
+:param-value: Iceland, Rekjavik
+
 .Example Query:
-[source,cypher,role=query-example,param-name=location,param-value="Iceland, Rekjavik",result-column=ip,expected-result=10.0.0.254]
+[source,cypher,role=query-example-params]
```
Comment (fbiville) [inline README.adoc:42]:
```suggestion
[source,cypher,role=query-example-params,subs=+attributes]
```
as per Guillaume's suggestion

**Review by fbiville — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (JMHReif) [reply README.adoc]: Fixed! Thank you for the feedback!

**Review by fbiville — APPROVED** [review]

**Comment by fbiville** [comment]
There seems to be conflicts, can you rebase your branch @JMHReif?

**Pull request merged**